### ISSUE TITLE
Link to download is broken

### ISSUE THREAD

**Issue by dinocaputo** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#6145]
https://aka.ms/TAPAACQAnalytics needs to be updated to actual download that is GA.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: cff4fb5a-c48b-4864-1bd7-3f933e2983ed
* Version Independent ID: f7e81551-61c6-3595-db92-59dfb1da5506
* Content: [Using CQD Power BI report to view Auto Attendant & Call Queue Historical Report - Microsoft Teams](https://docs.microsoft.com/en-us/microsoftteams/cqd-teams-aa-cq-historical-report)
* Content Source: [Teams/CQD-teams-aa-cq-historical-report.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/CQD-teams-aa-cq-historical-report.md)
* Service: **msteams**
* GitHub Login: @CLYVR
* Microsoft Alias: **colongma**

**Comment by CLYVR** [comment]
Note: Duplicate of 6144

Thanks for your message. The file that is linked to is the GA Power BI template file. I'm waiting on the final storage location for the file and will then be able to update the article however all that will change is the link, the file will be identical. I'll let you know when this occurs.

**Comment by nam31** [comment]
@dinocaputo This issue is being closed as this is a duplicate and the issue is already being addressed in the original post #6144
